src/mame/sun/v120.cpp

```diff
@@ -0,0 +1,159 @@
+#include "emu.h"
```

Comment:
This file is missing the license and copyright-holders headers. You should also put some basic information about the driver at the top.